app/models/user.rb

```diff
@@ -0,0 +1,35 @@
+class User < ApplicationRecord
+  include Authenticatable
+  include Steppable
+
+  enum aasm_state: {
```

Comment:
Store stages as an integer (Rails enum). Leave plenty of space in-between in case we ever had to add stages between existing ones. This would obviate the need for a data migration in that case.